server/build/webpack.js

```diff
@@ -171,4 +172,7 @@ export default async function createCompiler (dir, { hotReload = false } = {}) {
       return interpolateNames.get(this.resourcePath) || url
     }
   })
+  const config = getConfig(dir)
+  const userWebpackConfig = config.webpack(webpackConfig, hotReload)
```

Comment:
Do this only if user provides a webpack decorator.
If there's a one, we should log it so the user know this is happening.